src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -173,94 +192,88 @@ This rule contains three main components:
-
+        Finally, this rule defines the set of obfuscation actions to apply to the logs matching the filter. Each action defines:
+        
+        * Which previously created obfuscation expression to use to extract the sensitive information from each set of attributes
+        * Which obfuscation method (`HASH_SHA256` or `MASK`) to be applied to obfuscate this data
+      
+      
         Note that when defining obfuscation rules via the GraphQL API, you will need to specify the `id` of the obfuscation expressions instead of their names. To make the previous example more readable, we used the obfuscation expression names instead.
     </td>
-  </tr>
-  </tbody>
-</table>
-
-### 4. Reusing expressions in other rules [#example-more]
-
-As a final example, imagine we also needed to obfuscate logs coming from another service named "Checkout Service" that have an attribute `serviceName = checkoutService` as well as a `ccn` attribute that contains credit card information:
-
-```json
-{
-  "message": "Order completed",
-  "ccn": "4321-5678-9876-2345",
-  "department": "sales",
-  "serviceName": "checkoutService"
-}
-```
+    </tr>
+    </tbody>
+  </table>
+  
+  </Collapser>
 
-To obfuscate the logs from this service, we would only have to define another obfuscation rule targeting these specific logs, and we would simply reuse the previously created `Credit card number` obfuscation expression:
+  <Collapser
+    id="example-more"
+    title="4. Reusing expressions in other rules"
+  >
 
-```json
-{
-  "name": "Obfuscate Checkout Service Logs",
-  "filter": "serviceName = 'checkoutService' AND department = 'sales'",
-  "actions": [
-    {
-      "attributes": ["ccn"],
-      "expression": {"name": "Credit Card Number"},
-      "method": "HASH_SHA256"
-    }
-  ]
-}
-```
+  As a final example, imagine we also needed to obfuscate logs coming from another service named "Checkout Service" that have an attribute `serviceName = checkoutService` as well as a `ccn` attribute that contains credit card information:
+
+  ```json
+  {
+    "message": "Order completed",
+    "ccn": "4321-5678-9876-2345",
+    "department": "sales",
+    "serviceName": "checkoutService"
+  }
+  ```
+
+  To obfuscate the logs from this service, we would only have to define another obfuscation rule targeting these specific logs, and we would simply reuse the previously created `Credit card number` obfuscation expression:
+
+  ```json
+  {
+    "name": "Obfuscate Checkout Service Logs",
+    "filter": "serviceName = 'checkoutService' AND department = 'sales'",
+    "actions": [
+      {
+        "attributes": ["ccn"],
+        "expression": {"name": "Credit Card Number"},
+        "method": "HASH_SHA256"
+      }
+    ]
+  }
+  ```
+  
+  </Collapser>
+</CollapserGroup>
 
-## Checklist: steps to obfuscate logs [#checklist]
+## Checklist: Steps to obfuscate logs [#checklist]
 
 To obfuscate your logs:
 
 1. Study the shape of your logs:
```

Comment:
Since it's step 1, I might make this less of a passive action by clarifying what/how they should use to study. I'm not sure if the questions you listed were as helpful as just saying, 'Study the shape of your logs by identifying patterns of sensitive data that appear in them.'

(Alternatively it could say: "Study the shape of your logs using that handy 'How obfuscation works' section I so graciously wrote out above.") (just kidding!)

Reply:
agreed